plugins/modules/postgresql_set.py

```diff
@@ -318,7 +342,7 @@ def main():
     # Allow to pass values like 1mb instead of 1MB, etc:
     if value:
         for unit in POSSIBLE_SIZE_UNITS:
-            if value[:-2].isdigit() and unit in value[-2:]:
+            if (value[:-2].isdigit() and unit in value[-2:]) or ('b' in value[-1] and value[:-1].isdigit()):
```

Comment:
this checks for the single `b` each time. shouldn't that check be outside of the for loop?

Reply:
@k3rni yes, i'm in the business trip now, I'll rewrite the whole loop soon, I noticed that it's extra. Will fix it, thanks!

Reply:
Rewrote. I could've written one long condition but split it intentionally for better readability